Needed for HEAD and glibc-2_7-branch.

From Debian glibc repository.  Patch by Aurelien Jarno.

Reported on <http://sourceware.org/bugzilla/show_bug.cgi?id=5246#c4>.


r2650 of debian/patches/hurd-i386/submitted-strtoul.diff.

Date: 2007-11-02 09:01:08 +0000 (Fri, 02 Nov 2007)
New Revision: 2650

  * Add hurd-i386/submitted-strtoul.diff: new patch to use
    __strtoul_internal() instead of strtoul() in internal functions.

2007-11-02  Aurelien Jarno  <aurelien@aurel32.net>

	* hurd/lookup-retry.c (__hurd_file_name_lookup_retry): Use
	__strtoul_internal instead of strtoul.


---
 hurd/lookup-retry.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hurd/lookup-retry.c b/hurd/lookup-retry.c
index 0a88622..96968f8 100644
--- a/hurd/lookup-retry.c
+++ b/hurd/lookup-retry.c
@@ -194,7 +194,7 @@ __hurd_file_name_lookup_retry (error_t (*use_init_port)
 		  char *end;
 		  int save = errno;
 		  errno = 0;
-		  fd = (int) strtoul (&retryname[3], &end, 10);
+		  fd = (int) __strtoul_internal (&retryname[3], &end, 10, 0);
 		  if (end == NULL || errno || /* Malformed number.  */
 		      /* Check for excess text after the number.  A slash
 			 is valid; it ends the component.  Anything else
